.gitlab/build-deb-rpm.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+if [ -n "$CI_COMMIT_TAG" ] && [ -z "$PYTHON_PACKAGE_VERSION" ]; then
+  PYTHON_PACKAGE_VERSION=${CI_COMMIT_TAG##v}
+fi
+
+echo -n $PYTHON_PACKAGE_VERSION > auto_inject-python.version
+
+source common_build_functions.sh
+
+# Download the megawheel
+mkdir -p dd-trace.dir/lib
+
+# Install known compatible pip as default version shipped in Ubuntu (20.0.2)
+# does not work.
+python3 -m pip install -U "pip>=22.0"
+python3 -m pip install packaging
+
+echo `pwd`
+
+../lib-injection/dl_wheels.py \
```

Comment:
Please note that if this is building binary shared objects, you should carefully consider which distribution versions you want to support with the packages. For example, we support CentOS/RHEL 6 with the Agent, which has glibc 2.12. Based on what you're aiming to support with these packages, you might need to build on a very old builder to get the proper glibc compatibility.

Reply:
In talking to the team, it's okay that the python package won't support all OSes that the agent supports.